README.md

```diff
@@ -1,70 +1,64 @@
-# Getting Started with Create React App
+# math-magicians-react
```

Comment:
Great job @brainskev 🤝

- [ ] Great work on updating your readme file. However, you never used the correct readme template. Kindly use the correct [readme template](https://github.com/microverseinc/readme-template) and customize it accordingly.